pages/view/_iscnId/index.vue

```diff
@@ -241,7 +241,12 @@
             :label="$t('iscn.meta.id')"
             class="mb-[12px]"
           >
-            <Label :text="iscnId" tag="div" preset="p6" />
+            <div class="flex justify-start flex-nowrap gap-[8px]">
+              <Label :text="iscnId" tag="div" preset="p6" />
+              <div class="cursor-pointer" @click="handleClickCopy">
+                <IconCopy class="w-[14px]"/>
+              </div>
```

Comment:
Can we use `<Label/>` `append` slot for icon copy? Alos click entire `<Label/>` to copy